vms/avm/environment_test.go

```diff
@@ -110,13 +104,12 @@ type envConfig struct {
 }
 
 type environment struct {
-	genesisBytes  []byte
-	genesisTx     *txs.Tx
-	sharedMemory  *atomic.Memory
-	issuer        chan common.Message
-	vm            *VM
-	service       *Service
-	walletService *WalletService
```

Comment:
dropped service and walletService. Using txBuilder to create txs instead.
service and walletService survive in their own unit tests until we kill deprecated X-chain APIs